README.md

```diff
@@ -1,44 +1,140 @@
+
+**OAuth**
+
+  Visitors to ObituaryHub have the option to sign-up for an account and login using either Google or Facebook using omniauth. Google was relatively easy to setup compared to Facebook, whose documentation was challenging to parse. We ended up having to create different apps on their site: one for testing and one for production. This functionality was completed painstakingly by [Sage Lee](https://github.com/sagemlee). [Jack Cullen](https://github.com/jpc20) helped to get Google up and running.
+
 ### Challenges
 
+We had originally intended to make our website as friendly as possible to the elderly. To an extent, this was our imagined user. The further along we got in the project, we realized that we (4 millenials) were building something that we could imagine using and deriving benefit from. Since we were all very new to Front End Development, we struggled with getting basic functionality accomplished, let alone taking into account extra accessibility issues. We hope that someone will join our efforts in the future and help us to accomplish this goal.
+
+As for the more mundane, we had all the normal challenges with remote group work, but were able to rely on GitHub heavily for conversations about code. We had an extremely detailed project board and strict rules that all members must approve a pull requests, including extensive comments both by the person submitting the PR and other group members. This helped us catch errors and also get better acquainted with the code that each other was contributing to the project.
+
+This is still a work in progress and there are definitely still many bugs. For example, the table columns for search results aren't sized correctly and when you condense the window width-wise while on a show page the text overlaps with the image. Furthermore, we have a lot of functionality in our extensions that we weren't able to complete in the 10 days alotted for this project. That is why we have announced on [our website](https://obituary-hub.herokuapp.com/resources) that this project will now be open source and we are inviting others in the Turing community and beyond to make contributions.
+
 ## Tech Stack
 
  - Ruby 5.1.7
  - Rails 5.2.4.3
  - PostgreSQL
- - Gems
-   * [Fast JSON API](https://github.com/Netflix/fast_jsonapi)(Thanks Netflix!)
-   * [Rack CORS Middleware](https://github.com/cyu/rack-cors)
-   * [Factory Bot](https://github.com/thoughtbot/factory_bot_rails)
-   * [Faker](https://github.com/faker-ruby/faker)
+ - Sendgrid
+ - JQuery
+ - Heroku
+ - TravisCI
+ - Sinatra
+ - Omniauth
```

Comment:
Did I miss anything? Not sure about whether to list gems separately.

Reply:
Looks right to me! I feel like we can leave the gems out? There are so many.

Reply:
Maybe ActiveRecord?